### DISCUSSION TITLE
Welcome to Solidus Discussions!

### DISCUSSION THREAD

**Discussion by kennyadsl** [solidusio/solidus#3856] [New Features or Ideas]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪. Please, take a look at our [Community Guidelines](https://solidus.io/community-guidelines/) before posting.

To get started, comment below with an introduction of yourself and tell us about what you do with this community. 👇

**Comment by jarednorman** [comment]
👋🏻 My name is Jared! I've been working with Solidus since we had the idea to fork away from Spree. These days I run [Super Good](https://supergood.software) and am a member of Solidus's Core and Stakeholders teams.

I help out frequently in the Slack community so I'm very excited to use GitHub Discussions where everyone can benefit from answers to questions and solutions to problems people run into!

**Comment by seand7565** [comment]
Hey, I'm Sean! I'm a developer at [Nebulab](http://nebulab.it/) and member of the Stakeholder team. I also organize some of the Solidus events like the meetups & conferences. Reach out if you're interested in helping out or speaking at one of the events!
